### PR TITLE
Make clear that the README assumes you have the source.

### PR DESCRIPTION
**Pull request by dthkao**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly-bundled-snippets/33)

### CODE REVIEW

**Comment by adorokhine** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[README.md, line 13 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre0DvmD8KMhrxT-wU:-Kfre0DvmD8KMhrxT-wV:bvb6ur9) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L13)):*
> ```Markdown
> ## Usage
> 
> 1.  Get the source either by dowloading or cloning the project repo:
> ```

downloading

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre1Kf6mJT7zcl1HYB:-Kfre1Kf6mJT7zcl1HYC:b-b8na0o) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L15)):*
> ```Markdown
> 1.  Get the source either by dowloading or cloning the project repo:
> 
>         git clone git@github.com:google/mobly-bundled-snippets.git
> ```

non-contributors don't have access to clone through ssh as far as I know, only https

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33)*

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Note: this is not an official Google product.
 
 ## Usage
 
+1.  Get the source either by dowloading or cloning the project repo:
```
Comment (xpconanfan) [inline README.md]: only clone's cmd is shown.
maybe we can just list the clone option only

**Comment by dthkao** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre1Kf6mJT7zcl1HYB:-Kfreaq-VzYBnrcFwynK:brydsqu) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L15)):*
<details><summary><i>Previously, adorokhine (Alexander Dorokhine) wrote…</i></summary><blockquote>

non-contributors don't have access to clone through ssh as far as I know, only https
</blockquote></details>

This is a good point. How about "Get the source either by downloading or cloning the project, and navigate to the project root." No example shell command.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33)*

**Comment by xpconanfan** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre1Kf6mJT7zcl1HYB:-KfrexiHKADbvmBtzPsq:b-m63wl) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L15)):*
<details><summary><i>Previously, dthkao (David T.H. Kao) wrote…</i></summary><blockquote>

This is a good point. How about "Get the source either by downloading or cloning the project, and navigate to the project root." No example shell command.
</blockquote></details>

Is there a better way for people to use MBS instead of having to download the code and compile?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33)*

**Comment by adorokhine** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre1Kf6mJT7zcl1HYB:-KfrgSuDD3pcYp_4o7CX:b-k93cto) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L15)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

Is there a better way for people to use MBS instead of having to download the code and compile?
</blockquote></details>

Does github offer a way to manage prebuilt downloads?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33)*

**Comment by xpconanfan** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre1Kf6mJT7zcl1HYB:-KfrgoGWyk7ILgUEGhc9:bxy30fo) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L15)):*
<details><summary><i>Previously, adorokhine (Alexander Dorokhine) wrote…</i></summary><blockquote>

Does github offer a way to manage prebuilt downloads?
</blockquote></details>

Looks like it does through the Release tab.
E.g. this project https://github.com/pompobit/ScanGO/releases

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33)*

**Comment by dthkao** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33#-Kfre1Kf6mJT7zcl1HYB:-Kfy_7tQeiAb678ChR1X:b-zdhe8m) ([raw file](https://github.com/google/mobly-bundled-snippets/blob/4d54c2a82658ee91bf3370bdab14a39213b41563/README.md#L15)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

Looks like it does through the Release tab.
E.g. this project https://github.com/pompobit/ScanGO/releases
</blockquote></details>

Until we get that up, we'll still need users to build their own right?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly-bundled-snippets/33)*

**Pull request closed without merging**